sql/src/test/java/io/crate/executor/transport/AlterTableOperationTest.java

```diff
@@ -72,4 +70,27 @@ public void testPrepareRenameTemplateRequests() throws Exception {
 
         assertThat(requests.v2().name(), is(PartitionName.templateName(sourceIdent.schema(), sourceIdent.name())));
     }
+
+    @Test
+    public void testPutMappingRequesets() throws Exception {
```

Comment:
Could we make this method name a bit more concrete or add a short comment to clarify what aspect this is testing?